Sources/SourceKitLSP/SourceKitServer.swift

```diff
@@ -96,6 +98,7 @@ public final class SourceKitServer: LanguageServer {
     registerToolchainTextDocumentRequest(SourceKitServer.documentColor, [])
     registerToolchainTextDocumentRequest(SourceKitServer.colorPresentation, [])
     registerToolchainTextDocumentRequest(SourceKitServer.codeAction, nil)
+    registerToolchainTextDocumentRequest(SourceKitServer.callHierarchyPrepare, nil)
```

Comment:
I would prefer we don't register these dummy functions until we can actually test the implementation.  Once we resolve the questions I made about the LSP API types, I would be okay with committing those independently as long as any non-trivial serialization gets tests added.  But I think the server changes should wait for an implementation.

Reply:
Removed